src/experiments-service.ts

```diff
@@ -0,0 +1,109 @@
+import { transpileModule } from 'typescript';
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CONFIG } from './config';
+import { GoogleAdsApi } from './google-ads-api';
+
+export class ExperimentsService {
+  private readonly _googleAdsApi;
+  private _campaigns: string[];
+
+  constructor(private readonly _campaignsCsv: string) {
+    this._campaigns = this._campaignsCsv
+      .split(',')
+      .map(e => '' + parseInt(e.trim()))
```

Comment:
Why do we need to parseInt as well, since we are converting it into a string afterwards?

Wouldn't .map(e => e.trim()) do the same thing?

Reply:
This is just a simple check that user specified IDs and not some additional text.